samples/hello-oboe/src/main/cpp/HelloOboeEngine.cpp

```diff
@@ -132,30 +132,39 @@ void HelloOboeEngine::restart() {
 
 oboe::Result HelloOboeEngine::start() {
     std::lock_guard<std::mutex> lock(mLock);
```

Comment:
Is it ok to hold the lock the whole time with the new set of operations? This is probably fine, but I want just to check with you if you have any concerns

Reply:
It is a long time. But safe and probably OK. I am going to move all this into an AsyncTask later so it won't hang the UI thread.